src/requests/rocketchat.js

```diff
@@ -0,0 +1,194 @@
+const axios = require('axios')
+const apiEndpoints = require('@constants/endpoints')
+
+const chatPlatformAxios = axios.create({
+	baseURL: process.env.CHAT_PLATFORM_URL,
+	headers: {
+		'X-Auth-Token': process.env.CHAT_PLATFORM_ACCESS_TOKEN,
```

Comment:
@nevil-mathew make it configurable using env AUTH_TOKEN_HEADER_NAME
by default set X-Auth-Token. 

Please make the same changes  for X-User-Id as well.  it can be : USER_ID_HEADER_NAME